packages/utils-general/src/blockchain-facade/EnumExports.ts

```diff
@@ -15,3 +15,9 @@ export enum Unit {
     GW = 1e9,
     GWh = 1e9
 }
+
+export enum DemandStatus {
```

Comment:
Can this be moved to the exchange package?

Reply:
I though so too, but exchange-core does not have any notion of the demand, it's bids, asks, orders, trades. Demand is a collection of bids.